tests/std/tests/GH_001858_iostream_exception/test.cpp

```diff
@@ -229,6 +235,33 @@ void test_ostream_exceptions() {
             // Expected case
         }
     }
+
+    { // operator<< with exceptions
+        basic_ostream<CharT> os(buffer.to_buf());
+        os.exceptions(ios_base::goodbit);

```

Comment:
This is inconsistent. `exceptions()` defaults to `goodbit`, so we don't need to say that here. Above, there are pairs of test cases that say `// meow` (using the default behavior) and then `// meow with exceptions` where `exceptions()` is called. I think we should follow that convention here.

For extra clarity, we could cite #4322.